lib/perl/Genome/WorkflowBuilder/Converge.pm

```diff
@@ -73,4 +74,37 @@ sub notify_output_link {
     return;
 }
 
+sub get_ptero_builder_task {
+    require Ptero::Builder::Detail::Workflow::Task;
+
+    my $self = shift;
+
+    $self->validate;
+
+    my %params = (
+        name => $self->name,
+        methods => [
+            $self->_get_ptero_converge_method(),
+        ],
+    );
+    if (defined $self->parallel_by) {
+        $params{parallel_by} = $self->parallel_by;
+    }
+    return Ptero::Builder::Detail::Workflow::Task->new(%params);
+}
+
+sub _get_ptero_converge_method {
+    require Ptero::Builder::Detail::Workflow::Converge;
+
+    my $self = shift;
+    my @output_names = $self->output_properties;
+    return Ptero::Builder::Detail::Workflow::Converge->new(
+        name => 'converge',
+        parameters => {
+            input_names => [$self->input_properties],
+            output_name => shift @output_names,
```

Comment:
Will there only ever be one of these? If that's the case, does it make more sense to do something like `my ($output_name) = $self->output_properties()` instead of putting it in an array and immediately shifting it off?